packages/components/src/internal/components/forms/FormStep.tsx

```diff
@@ -152,7 +152,7 @@ export const withFormSteps = (Component: any, defaultState?: WithFormStepsState)
             this.selectStep = this.selectStep.bind(this);
 
             this.state = {
-                currentStep: defaultState && defaultState.currentStep !== undefined ? defaultState.currentStep : 1,
+                currentStep: props.initialStep ? props.initialStep : (defaultState && defaultState.currentStep !== undefined ? defaultState.currentStep : 1),
```

Comment:
The typings here are really throwing me for a loop.  I believe that this works, but can't find where this `initialStep` prop comes from.  Doesn't look new necessarily, so I'm surely missing something, but please enlighten me.

Reply:
Sorry for the confusion. The `initialStep` was a new prop. I has just forgotten to add it to the type definition. Done.